### PR TITLE
GG-19277

### PR DESCRIPTION
**Pull request by denis-chudov**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java [hunk 1]
```diff
@@ -152,11 +126,48 @@ public static void main(String[] args) {
         System.exit(hnd.execute(Arrays.asList(args)));
     }
 
+    /**
+     * @return prepared JULs logger.
+     */
+    private static Logger setupJavaLogger() {
+        Logger result;
```
Comment (denis-chudov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: You can do it in 1 line.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java [hunk 2]
```diff
@@ -152,11 +126,48 @@ public static void main(String[] args) {
...
+
+        result = Logger.getLogger(CommandHandler.class.getName() + "Log");
+        result.setLevel(Level.INFO);
+        result.setUseParentHandlers(false);
```
Comment (denis-chudov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: Please add empty lines in accordance with codestyle.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java [hunk 3]
```diff
@@ -152,11 +126,48 @@ public static void main(String[] args) {
...
+
+        // Adding logging to file.
+        try {
+            String absPathPattern = new File(JavaLoggerFileHandler.logDirectory(U.defaultWorkDirectory()), "control-utility-%g.log").getAbsolutePath();
+
+            FileHandler fileHandler = new FileHandler(absPathPattern, 5 * 1024 * 1024, 5);
+            fileHandler.setFormatter(new JavaLoggerFormatter());
```
Comment (denis-chudov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: Please add empty lines in accordance with codestyle.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java [hunk 4]
```diff
@@ -271,6 +284,10 @@ public int execute(List<String> rawArgs) {
             logger.info("Command [" + commandName + "] finished with code: " + EXIT_CODE_UNEXPECTED_ERROR);
 
             return EXIT_CODE_UNEXPECTED_ERROR;
+        } finally {
+           Arrays.stream(logger.getHandlers())
+                 .filter(handler -> handler instanceof FileHandler)
+                 .forEach(Handler::close);
```
Comment (denis-chudov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: Please fix indentation here.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java [hunk 5]
```diff
@@ -152,11 +126,48 @@ public static void main(String[] args) {
...
+            System.out.println("Failed to configure logging to file");
+        }
+
+        // Adding logging to console.
+        result.addHandler(setupStreamHandler());
+
+        return result;
+    }
+
+    public static StreamHandler setupStreamHandler() {
```
Comment (denis-chudov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: Please add javadoc for this method.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java (hunk 1 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java (hunk 2 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java (hunk 3 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java (hunk 4 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java (hunk 5 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java]: fixed

**Comment by dspavlov** [comment]
The ticket is resolved, closing PR.  
Feel free to reopen this PR if it is anyway actual.

**Pull request closed without merging**